src/node/flows.ml

```diff
@@ -9,6 +9,40 @@ let write_state_to_file path protocol =
       Lwt_io.with_file ~mode:Output path (fun oc ->
           let%await () = Lwt_io.write oc protocol_bin in
           Lwt_io.flush oc))
+
+let string_of_error = function
+  | `Added_block_has_lower_block_height -> "Added block has lower block height"
```

Comment:
Should the error printed be more `human friendly`?

Reply:
Resolving this, you're right, but there is an open issue for it. https://github.com/marigold-dev/deku/issues/64